src/types.json

```diff
@@ -0,0 +1,79 @@
+{
+  "Terms of service": {
+    "commitment": {
+      "writer": "service provider",
+      "audience": "end user",
+      "object": "service usage"
+    }
+  },
+  "Privacy Policy": {
+    "commitment": {
+      "writer": "service provider",
+      "audience": "end user",
+      "object": "personal data"
+    }
+  },
+  "Developer Agreement": {
+    "commitment": {
+      "writer": "service provider",
+      "audience": "developer",
+      "object": "API and Content usage"
+    }
+  },
+  "Developer Policy": {
```

Comment:
What's the difference with `Developer Agreement`?

Reply:
I didn't find one. But Twitter define both.
If you can explain the difference:
https://developer.twitter.com/fr/developer-terms/agreement
https://developer.twitter.com/fr/developer-terms/policy

Reply:
Thanks.
I feel that the “agreement” is a full, legally binding contract, whereas the “policy” describes what Twitter expects and how it can be expected to behave.

I find this separation very interesting, but a bit uncommon as well.

I see two options, in order of preference:

1. Rewrite the `object` of the Developer Policy as “expected behavior of and towards third-party services”.
2. Stop tracking the Developer Policy for Twitter as of now, since we'll need to cover the specific case of Twitter and its multitude of documents in some special way anyway.

I'll let you decide.

Reply:
If we keep both, which one will you use for theses documents?
- https://developers.facebook.com/policy
- https://help.instagram.com/325135857663734
- https://play.google.com/intl/ALL_en/about/developer-distribution-agreement.html

Reply:
Okay, it's a mess 🙃

In favor of option 2 then.

Reply:
Ok, if we keep one, which one do you prefer, "Developer Terms", "Developer Policy", "Developer Agreement"?

Reply:
“Policy” seems more common, isn't it? Otherwise I'd say “Developer Terms”, by similarity with “Terms of Service“.

Reply:
Yep, I also found that "Policy" seems more common but as you explained for Twitter where “agreement” is a full, legally binding contract, whereas the “policy” describes what Twitter expects and how it can be expected to behave, "agreement" seems to be more in line with what we want. 
Another point, if we keep "Policy", for Twitter the url is not the one which target Twitter Policy but this one "https://developer.twitter.com/developer-terms/agreement" which target the agreement.
So, even if "Developer Terms" seems less common, I think it avoids confusion while still being very understandable (and similar with “Terms of Service“ as you said), so I vote for this one.